packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -276,6 +276,14 @@ interface DatabaseClusterBaseProps {
    */
   readonly storageEncryptionKey?: kms.IKey;
 
+  /**
+   * The storage type to be associated with the DB cluster.
+   * Valid values: `aurora` , `aurora-iopt1` (Aurora DB clusters); `io1` (Multi-AZ DB clusters)
```

Comment:
```suggestion
```

The valid values will come from the enum.

Reply:
Updated